docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -0,0 +1,101 @@
+= Set up the Observability AI Assistant
+
+Complete the following steps to use the Observability AI Assistant:
+
+. Create an API key with your AI provider to authenticate requests from the Security Assistant. You'll use this in the next step. Refer to the provider's documentation for generating API keys:
++
+* https://platform.openai.com/docs/api-reference[OpenAI]
+* https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
+
+. Update {kib}'s configuration settings to include the Observability AI Assistant feature flag and additional information about your AI set up.
++
+.. Navigate to the {kib} configuration settings according to your deployment:
++
+** *Self-managed (on-premises) deployments*: Add the configuration to the `kibana.yml` file, which is used to {kibana-ref}/settings.html[configure {kib}].
+** *{ecloud} deployments*: Use the YAML editor in the {ecloud} console to add the configuration to {cloud}/ec-manage-kibana-settings.html[{kib} user settings].
++
+.. Add one of the following configurations that corresponds the AI service you're using:
++
+** *OpenAI*:
++
+In the following example, `xpack.observability.aiAssistant.provider.openAI.apiKey` is the API key you generated in the first step.
++
+[source,yaml]
+----
+xpack.observability.aiAssistant.enabled: true
+#
+xpack.observability.aiAssistant.provider.openAI.apiKey: xxxx
+# 
+xpack.observability.aiAssistant.provider.openAI.model: gpt-4
+----
++
+** *Azure OpenAI Service*:
++
+In the following example, `xpack.observability.aiAssistant.provider.azureOpenAI.apiKey` is the API key you generated in the first step, and `xpack.observability.aiAssistant.provider.azureOpenAI.resourceName` is the name of the Azure OpenAI resource used to deploy your model. 
++
+[source,yaml]
+----
+xpack.observability.aiAssistant.enabled: true
+#
+xpack.observability.aiAssistant.provider.azureOpenAI.deploymentId: gpt-4
+# 
+xpack.observability.aiAssistant.provider.azureOpenAI.resourceName: xxxx
+# 
+xpack.observability.aiAssistant.provider.azureOpenAI.apiKey: xxxx
+----
+
+. If you're using a self-managed deployment, restart {kib}.
+
+[discrete]
+[[obs-ai-interact]]
+= Interact with the Observability AI Assistant
+
+You can find Observability AI Assistant prompts throughout {observability} that provide the following information:
+
+- *Universal Profiling* – explains the most expensive processes and functions in your fleet and provides optimization suggestions.
+- *Application performance monitoring (APM)* – explains APM errors and provides remediation suggestions.
+- *Infrastructure Observability* – explains host process and provides suggestions for optimizing CPU usage.
+- *Logs* – explains log messages and generates search patterns to find similar issues.
+- *Alerting* – provides possible log spike causes and remediation suggestions.
+
+
+For example, in the logs details, you'll see prompts for *What's this message?* and *How do I find similar log messages*: 
```

Comment:
Is "logs details" right here? "Log details" sounds better unless there's some precedent for making logs plural.